openapi/index.yaml

```diff
@@ -0,0 +1,372 @@
+          description: Profile created
+          schema:
+            $ref: "#/definitions/ExtendedProfile"
+        "400":
+          description: Invalid request.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized
+        "429":
+          description: Too many requests
+    put:
+      operationId: updateProfile
+      summary: UpdateProfile
+      description: |-
+        Update a user profile
+      tags:
+        - restricted
+      parameters:
+        - $ref: "#/parameters/FiscalCode"
+        - in: body
+          name: body
+          schema:
+            $ref: "#/definitions/Profile"
+      responses:
+        "200":
+          description: Profile updated
+          schema:
+            $ref: "#/definitions/ExtendedProfile"
+        "400":
+          description: Invalid request.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized
+        "429":
+          description: Too many requests
+  "/profiles/{recipient}/sender-services":
+    get:
+      operationId: getServicesByRecipient
+      summary: GetServicesForRecipient
+      description: |-
+        Returns the service IDs of all the services that have contacted the recipient,
+        identified by the provided fiscal code, at least once.
+      tags:
+        - restricted
+      parameters:
+        - name: recipient
+          in: path
+          type: string
+          required: true
+          description: The recipient's fiscal code.
+          minLength: 16
+          maxLength: 16
+          pattern: "[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]"
+      responses:
+        "200":
+          description: Found.
+          schema:
+            $ref: "#/definitions/PaginatedServiceTupleCollection"
+        "401":
+          description: Unauthorized
+        "429":
+          description: Too many requests
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  SubscriptionKey:
+    type: apiKey
+    name: Ocp-Apim-Subscription-Key
+    in: header
+    description: The API key obtained through the developer portal.
+parameters:
+  FiscalCode:
+    name: fiscal_code
+    in: path
+    type: string
+    maxLength: 16
+    minLength: 16
+    required: true
+    description: The fiscal code of the user, all upper case.
+    pattern: "[A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]"
+    x-example: SPNDNL80R13C555X
+  PaginationRequest:
+    name: cursor
+    in: query
+    type: string
+    minimum: 1
+    description: An opaque identifier that points to the next item in the collection.
+    x-example: 01BX9NSMKVXXS5PSP2FATZM123
+definitions:
+  ExtendedProfile:
+    $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/168821506-profile-email-validation/openapi/definitions.yaml#/ExtendedProfile"
```

Comment:
Temp reference to a specific branch until the PR of io-functions-commons will be merged.